kubeflow-pipelines/GHSA-4hjh-wcwx-xvwj.patch

```diff
@@ -1,47 +0,0 @@
-diff --git a/frontend/server/package-lock.json b/frontend/server/package-lock.json
-index 612cf0669..fe841dfa3 100644
---- a/frontend/server/package-lock.json
-+++ b/frontend/server/package-lock.json
-@@ -8,7 +8,7 @@
-         "@aws-sdk/credential-providers": "^3.621.0",
-         "@google-cloud/storage": "^2.5.0",
-         "@kubernetes/client-node": "^0.12.1",
--        "axios": ">=1.8.2",
-+        "axios": ">=1.12.0",
-         "crypto-js": "^3.1.8",
-         "express": "^4.21.0",
-         "gunzip-maybe": "^1.4.1",
-diff --git a/frontend/server/package.json b/frontend/server/package.json
-index 282d0c401..86c1cd737 100644
---- a/frontend/server/package.json
-+++ b/frontend/server/package.json
-@@ -5,8 +5,8 @@
-     "@aws-sdk/credential-providers": "^3.621.0",
-     "@google-cloud/storage": "^2.5.0",
-     "@kubernetes/client-node": "^0.12.1",
--    "axios": ">=1.8.2",
--    "crypto-js": "^3.1.8",
-+    "axios": ">=1.12.0",
-+    "crypto-js": "^4.2.0",
-     "express": "^4.21.0",
-     "gunzip-maybe": "^1.4.1",
-     "http-proxy-middleware": "^0.18.0",
-@@ -38,7 +38,17 @@
-   "overrides": {
-     "express": {
-       "path-to-regexp": "0.1.12"
--    }
-+    },
-+    "retry-request": {
-+      "debug": "^4.3.1"
-+    },
-+    "xml2js": "^0.5.0",
-+    "json-bigint": "^1.0.0",
-+    "tough-cookie": "^4.1.3",
```

Comment:
i think we still gonna need this npm module upgrade for a CVE GHSA-72xf-g2v4-qvf3.

```
└── 📄 /server/node_modules/tough-cookie/package.json
        📦 tough-cookie 2.5.0 (npm)
            Medium CVE-2023-26136 GHSA-72xf-g2v4-qvf3 fixed in 4.1.3
```